### ISSUE TITLE
When i enter the Prison of Bloodhound Knight Darriwil i get Random Runes

### ISSUE THREAD

**Issue by HerbertPiet** [uberhalit/EldenRingFpsUnlockAndMore#48]
Its like 1090 runes just on entering the prison. May or may not happen in other places.

I dont have "Disable Runes loss on death" activated and i tested it with and without the tool. Would kill this otherwise great tool for me as i just want the FPS while not influencing the true experience of the game.

Cheers!

**Comment by uberhalit** [comment]
Hello, 
are you sure this isn't default behaviour of the game? The rune loss code was adjusted with version 1.0.0.0 but if you never enabled it in the first place then there is hardly any reason runes should be affected by frame rate.

**Comment by HerbertPiet** [comment]
Yeah am pretty sure its not intended. I tested with/without anticheat and also your tool. I thought maybe there was a part of the rune code affecting it. Neverless, thanks for the great tool, i will be looking out for the newer versions and just try again :)

**Comment by dJani97** [comment]
I've been playing a lot with this tool and didn't notice anything like this.

I'm pretty sure some enemy is getting killed or falling to it's death in your game somewhere. Maybe the modifications cause NPC(s) in that specific area to clip through the terrain and die?
